tests/server/test_server.py

```diff
@@ -102,7 +102,7 @@ def storage_controller_fixture() -> Generator[MagicMock, None, None]:
 def fetch_certificates_fixture() -> Generator[MagicMock, None, None]:
     """Return a mocked fetch certificates."""
     with patch(
-        "matter_server.server.device_controller.fetch_certificates", autospec=True
+        "matter_server.server.server.fetch_certificates", autospec=True
```

Comment:
?

Reply:
In theory, we could run two Device controller (mutiple fabrics). But even in that case, we would only want to fetch once. So I've moved the PAA certificate fetching to the server. I think it makes more sense there, as this is a "server wide" problem. No need to clutter the device controller with it.

Reply:
yeah, but there is two time server now - shouldnt that be just matter_server.server.fetch_certificates

Reply:
ah no, ignore me this is module level ofc